.github/workflows/live-links.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
             http://www.adlinktech.com/en/index
```

Comment:
There seem to be other plain text URLs on the ignore list, that are also not valid regexp - do they work just fine?

Reply:
Yeah, this is to specify ignore that URL but in this case, we have sunbeam01, sunbeam02 and sunbeam03 but all example.com URLs should be ignored.

Reply:
OK. I guess these full URLs work a bit by accident, because they are valid Python regexp. I guess only `.` in these can is interpreted as any char, not as a `.`.  But for the purpose of ignoring it should be fine.